src/Components/Buttons/CancelButton.js

```diff
@@ -0,0 +1,9 @@
+import { Button } from "@mui/material"
+const CancelButton = (props) => {
+    return (
+        <>
+         <Button onClick={()=>props.setEdit(false)}>cancel</Button>
+        </>
+    )
+}
+export default CancelButton
```

Comment:
these componants are not really dont anything please split your code in a different way

Reply:
by components, i mean CancelButton, DeleteButton and EditButton